soundata/datasets/urbansound8k.py

```diff
@@ -0,0 +1,355 @@
+
+
+    Feedback
+    --------
+
+    Please help us improve UrbanSound8K by sending your feedback to: justin.salamon@nyu.edu
+    In case of a problem report please include as many details as possible.
+
+"""
+
+import os
+from typing import BinaryIO, Optional, TextIO, Tuple
+
+import librosa
+import numpy as np
+import csv
+
+from soundata import download_utils
+from soundata import jams_utils
+from soundata import core
+from soundata import annotations
+from soundata import io
+
+
+BIBTEX = """
+@inproceedings{Salamon:UrbanSound:ACMMM:14,
+	Address = {Orlando, FL, USA},
+	Author = {Salamon, J. and Jacoby, C. and Bello, J. P.},
+	Booktitle = {22nd {ACM} International Conference on Multimedia (ACM-MM'14)},
+	Month = {Nov.},
+	Pages = {1041--1044},
+	Title = {A Dataset and Taxonomy for Urban Sound Research},
+	Year = {2014}}
+"""
+REMOTES = {
+    "all": download_utils.RemoteFileMetadata(
+        filename="UrbanSound8K.tar.gz",
+        url="https://zenodo.org/record/1203745/files/UrbanSound8K.tar.gz",
+        checksum="9aa69802bbf37fb986f71ec1483a196e",
+        destination_dir="urbansound8k",
+    )
+}
+
+LICENSE_INFO = "Creative Commons Attribution Non Commercial 4.0 International"
+
+
+class Clip(core.Clip):
+    """urbansound8k Clip class
+
+    Args:
+        clip_id (str): clip id of the clip
+
+    Attributes:
+        audio_path (str): path to the audio file
+        slice_file_name (str): The name of the audio file. The name takes the following format: [fsID]-[classID]-[occurrenceID]-[sliceID].wav, where:
+            [fsID] = the Freesound ID of the recording from which this excerpt (slice) is taken
+            [classID] = a numeric identifier of the sound class (see description of classID below for further details)
+            [occurrenceID] = a numeric identifier to distinguish different occurrences of the sound within the original recording
+            [sliceID] = a numeric identifier to distinguish different slices taken from the same occurrence
+        fsID (str): The Freesound ID of the recording from which this excerpt (slice) is taken
+        start (float): The start time in seconds of the slice in the original Freesound recording
+        end (float): The end time in seconds of slice in the original Freesound recording
+        salience (int): A (subjective) salience rating of the sound. 1 = foreground, 2 = background.
+        fold (int): The fold number (1-10) to which this file has been allocated.
+        classID (int): A numeric identifier of the sound class:
+            0 = air_conditioner
+            1 = car_horn
+            2 = children_playing
+            3 = dog_bark
+            4 = drilling
+            5 = engine_idling
+            6 = gun_shot
+            7 = jackhammer
+            8 = siren
+            9 = street_music
+        class (str): The class name: air_conditioner, car_horn, children_playing, dog_bark, drilling, engine_idling, gun_shot, jackhammer, siren, street_music.
```

Comment:
Same here

Reply:
I've updated this docstring quite a bit please have another look once the PR is ready for another review